### ISSUE TITLE
bio.acousti.ca not working

### ISSUE THREAD

**Issue by edwbaker** [NaturalHistoryMuseum/scratchpads2#6490]
Server seems to be down.

**Comment by therobyouknow** [comment]
Good morning @edwbaker 

It looks like bio.acousti.ca is currently unreachable.  I pinged it:

```
robd@nhm-b3zj7y2-lnx:~$ ping bio.acousti.ca
ping: bio.acousti.ca: Name or service not known
```
I have checked the domain via whois and it looks OK - expiration next year etc.

Another reason for this could be the server itself not running.  Do you have more information on the server setup?  I am of the understanding that  bio.acousti.ca is on a setup separate from *.myspecies.info . NHM hosts sides with *.myspecies.info and a check on one of these just now shows it to be up.

Have you observed the site to be running between 7 December and 10 December - just before you raised the issue? 

I ask because we did a scratchpads release between 2 December and 6 December, which did have a performance impact on sites during this time but there some self-recovery https://github.com/NaturalHistoryMuseum/scratchpads2/issues/6488

I would expect the same of your site if you had the same release deployed, that things could start to be getting back to normal. 

In other words, before you raised this issue on 10 December, when was the last time before that, that you saw your site running.

**Comment by therobyouknow** [comment]
Just making enquiries. The release (  https://github.com/NaturalHistoryMuseum/scratchpads2/issues/6488 ) hasn't yet been applied to bio.acousti.ca so would not be the reason for the outage. Investigating...

**Comment by edwbaker** [comment]
It's on its own server but managed by NHM so I can't really help anymore than that. I assume Ben can advise on server details.

**Comment by therobyouknow** [comment]
Now back online - https://bio.acousti.ca/

![image](https://user-images.githubusercontent.com/1080646/146806652-c6bef6fc-a9fa-428a-81fc-d4e84ec61607.png)

Closing this ticket for now. I don't have any details as to how the site came to be back online, at the moment.
